docs/es/getting_started/create.md

```diff
@@ -1,22 +1,22 @@
 # Crea tu fork
 
-El repositorio git de Consul Democracy está hospedado en Github.com, recomendamos lo uses para tu fork para hacer las cosas mas fáciles. Pero puedes usar cualquier otro servicio como Bitbucket o Gitlab si deseas, pero no te olvides de poner el enlace en el footer a tu repositorio en cumplimiento con la licencia de este proyecto (GPL Affero 3).
+El repositorio git de Consul Democracy está hospedado en Github.com, recomendamos lo uses para tu fork para hacer las cosas mas fáciles. Pero puedes usar cualquier otro servicio como Bitbucket o Gitlab si deseas, pero no te olvides de poner el enlace en el footer a tu repositorio en cumplimiento con la licencia de este proyecto (AGPLv3).
```

Comment:
```suggestion
El repositorio git de Consul Democracy está hospedado en Github.com, que recomendamos que uses para tu fork para hacer las cosas más fáciles. Pero puedes usar cualquier otro servicio como Bitbucket o Gitlab si deseas; únicamente, no te olvides de poner el enlace en el pie de página a tu repositorio en cumplimiento con la licencia de este proyecto (AGPLv3).
```

The last one ("únicamente") is based on the English version; having two "pero" in the same sentences sounded a bit weird :thinking:.